src/util/segmentation/brush/labelmapStats.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * LabelmapStats - Returns the statistics of the requested labelmap.
+ *
+ * @param  {type} labelmapBuffer The buffer for the labelmap.
+ * @param  {Number[][]} imagePixelData The pixeldata of each image slice.
+ * @param  {Number} sliceLength    The number of pixels in one slice.
+ * @param  {Number} segmentIndex   The index of the segment.
+ * @returns {Promise} A promise that resolves to the stats.
+ */
+export default function labelmapStats(
```

Comment:
Is this a `util` because we expect to export it and use as general purpose? If not, it could be a private method for the segmentation module? Or registered by the segmentation module so that the logic can live alongside the code in the lib that uses it.

Reply:
Yeah, I don't see any reason for it to not just be part of the segmentation module.